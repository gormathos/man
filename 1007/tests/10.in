-66;-33;0;0;9;-60;42;78;6;-10;84;-89;-14;96;-89;-64;85;-36;79;65
-23;76;0;0;9;-91;-48;3;6;-38;-22;80;39;-85;-70;-96;70;-32;73;91
-60;45;0;0;9;-25;-17;41;6;-56;-50;94;-7;23;-75;27;59;-79;-8;-98
-59;88;0;0;9;56;54;-96;6;-70;24;-56;1;-68;13;-77;-14;-98;-34;-78
97;-93;0;0;9;88;-81;-39;6;-25;53;73;-3;74;18;22;88;27;0;78
75;69;0;0;9;84;37;80;6;4;-11;73;73;-26;-19;-51;33;84;5;-34
62;62;0;0;9;-74;-63;57;6;10;50;-1;-92;89;-26;-87;-53;-48;32;29
45;-97;0;0;9;97;-99;-64;6;-16;14;28;-9;81;-52;-48;-12;78;-64;-3
32;-59;0;0;9;86;87;-73;6;53;10;-41;96;55;-82;-82;-6;35;-94;14
15;-65;0;0;9;-75;-54;91;6;-52;-98;95;-94;-58;72;-100;-39;36;-78;19
-8;-31;0;0;9;-10;26;73;6;-35;45;99;26;-71;91;-9;19;-33;-51;76
