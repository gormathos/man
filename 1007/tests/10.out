-66;-33;-60;42;78;-10;84;-89;-14;96;-89;-64;85;-36;79;65
-23;76;-91;-48;3;-38;-22;80;39;-85;-70;-96;70;-32;73;91
-60;45;-25;-17;41;-56;-50;94;-7;23;-75;27;59;-79;-8;-98
-59;88;56;54;-96;-70;24;-56;1;-68;13;-77;-14;-98;-34;-78
97;-93;88;-81;-39;-25;53;73;-3;74;18;22;88;27;0;78
75;69;84;37;80;4;-11;73;73;-26;-19;-51;33;84;5;-34
62;62;-74;-63;57;10;50;-1;-92;89;-26;-87;-53;-48;32;29
45;-97;97;-99;-64;-16;14;28;-9;81;-52;-48;-12;78;-64;-3
32;-59;86;87;-73;53;10;-41;96;55;-82;-82;-6;35;-94;14
15;-65;-75;-54;91;-52;-98;95;-94;-58;72;-100;-39;36;-78;19
-8;-31;-10;26;73;-35;45;99;26;-71;91;-9;19;-33;-51;76
